notebooks/en/llm_gateway_pii_detection.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>Could also use the <code>&gt; [!TIP]</code> for the "Note" here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>hopefully addressed</p>